lib/datasource/remote/api/proposals_repository.dart

```diff
@@ -235,6 +235,29 @@ class ProposalsRepository extends NetworkRepository with EosRepository {
         .catchError((error) => mapHttpError(error));
   }
 
+  Future<Result> getDelegator(String account, String voiceScope) {
+    print('[http] get delegate for $account');
+
+    final request = createRequest(
+      code: accountFunds,
+      indexPosition: 2,
+      scope: voiceScope,
+      table: tableDelegates,
+      lowerBound: account,
+      upperBound: account,
+      limit: 100,
```

Comment:
generally at some point we will have to figure out what to do is the result has a more = true in it. this is just a comment.